app/controllers/users/two_step_verifications_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Users::TwoStepVerificationsController < ApplicationController
+  def new
+    redirect_to two_step_verification_path if current_user.two_factor_auth_enabled?
```

Comment:
Maybe this route should be called `two_step_authorisation_path`? Now I had to look up where this redirects to and why.